types/queries/get_bookmarked_on.sql

```diff
@@ -0,0 +1,3 @@
+SELECT bookmarked_on, document_id, user_id
```

Comment:
I wonder if `get_document_bookmarked_on` is a good name? I know that's even longer, but I think that expresses the intent of this query. If we have a document and want to know when a user bookmarked it, here is how we find out.

Reply:
I'll change it to `get_document_bookmarked_on` since it is indeed the intent of the loader as a whole. The reason the query needs to return all columns in the table is so it can properly insert it into the HashMap type required as a return for Loaders.